### PR TITLE
feat: updated the account drawer

### PR DESCRIPTION
**Pull request by impelcrypto**
**Pull Request Summary**

* feat: updated the account drawer
  * ticket: https://www.notion.so/astarnetwork/Wallet-drawer-improvement-sort-by-balance-6667df67247a42b5b3a61f374fd31b9e 

**Check list**
- [ ] contains breaking changes
- [x] adds new feature
- [ ] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [x] tested on mobile devices

**This pull request makes the following changes:**

**Changes**
* added `Show balance` toggle
* sorted the account order by balance

=before=
image

=after=
image

image

### CODE REVIEW

File: src/hooks/transfer/useTransfer.ts [hunk 1]
```diff
@@ -116,12 +116,9 @@ export function useTransfer(selectUnit: Ref<string>, decimal: Ref<number>, fn?:
       }
 
       const receivingAddress = isValidEvmAddress(toAddress) ? toSS58Address(toAddress) : toAddress;
-
-      watchEffect(async () => {
```
Comment (impelcrypto) [inline src/hooks/transfer/useTransfer.ts:120]: @sirius651 
I've removed `watchEffect` here. Please advise if it causes any unexpected behavior.

**Comment by Kahonnohak** [comment]
Looking lovely thanks.

Small requests please.

- The green connection and the radio buttons are squashed a bit. Please take a look.
- Would it be possible to change the colour of the toggle when `off`? Dark for `#5F656F`, Light for `#fff`
- Could you make the icon size (copy/link) `24px`Please?  32px was too big and would like to change.
- Could you make the padding before the button to `24px` ? This is also an existing minor improvement I wish to fix.

Thanks!

**Comment by impelcrypto** [comment]
@Kahonnohak 
Fixed. Please have a look.

=Before=
image

=After=
image

**Comment by Kahonnohak** [comment]
Thank you!

The toggle on the dark theme doesn't look off when it's white, so please make this change. Otherwise all good:)
Screenshot 2022-06-15 at 15 07 52

**Pull request merged**